.gitignore

```diff
@@ -87,3 +87,6 @@ cpp/Brewfile.lock.json
 java-dist/
 java-native-c/
 java-native-cpp/
+
+# files altered by build
+java/flight/flight-jdbc-driver/src/main/resources/properties/flight.properties
```

Comment:
This file seems to get generated each maven build

Reply:
Note https://github.com/apache/arrow/pull/13800 is the up-to-date PR and some of these things were addressed there